"make test" fails for me

`make test` fails for me on 72e0781277eac74eac364b2851a2d24e7f67e3c0:
```
cd cmd/ipfs && go build -i -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=72e0781"
go test ./...
core/commands/incfusever/incfusever.go:6:2: cannot find package "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version" in any of:
        /usr/local/go/src/github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version (from $GOROOT)
        /home/kevina/gocode/src/github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version (from $GOPATH)
```
`make TEST_NO_FUSE=1 test` seams okay

I think this relates to commit e79e1d31e9e0f318542746e81b9509673c20e433

Sorry if I am missing something obvious.

--- a/core/commands/incfusever/doc.go
+++ b/core/commands/incfusever/doc.go
@@ -1,7 +1,0 @@
-// Package incfusever is only here to prevent go src tools (like godep)
-// from thinking fuseversion is not a required package. Though we do not
-// actually use github.com/jbenet/go-fuse-version as a library, we
-// _may_ need its binary. We avoid it as much as possible as compiling
-// it _requires_ fuse headers. Users must be able to install go-ipfs
-// without also installing fuse.
-package incfusever

--- a/core/commands/incfusever/incfusever.go
+++ b/core/commands/incfusever/incfusever.go
@@ -1,9 +1,0 @@
-// +build !nofuse
-
-package incfusever
-
-import (
-	fuseversion "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version"
-)
-
-var _ = fuseversion.LocalFuseSystems

